src/pages/Settings/Network.tsx

```diff
@@ -106,7 +107,7 @@ const Network = (): JSX.Element => {
         opts: {
           masterPublicKey: masterPubKey,
           masterBlindingKey: masterBlindKey,
-          baseDerivationPath: network === 'regtest' || network === 'testnet' ? "m/84'/1'/0'" : "m/84'/0'/0'",
+          baseDerivationPath: network === 'liquid' ? BASE_DERIVATION_PATH_MAINNET : BASE_DERIVATION_PATH_NOT_MAINNET,
```

Comment:
Best to mark it already as mainnet but legacy, and anything is non mainnet is fine to call it testnet
```suggestion
          baseDerivationPath: network === 'liquid' ? BASE_DERIVATION_PATH_LIQUID_LEGACY : BASE_DERIVATION_PATH_TESTNET
```